test/ui/segmented-bar/segmented-bar-test.html

```diff
@@ -0,0 +1,144 @@
+                }
+            },
+            "segment2": {
+                "prototype": "montage/ui/segment.reel",
+                "properties": {
+                    "element": {"#": "segment2"}
+                },
+                "bindings": {
+                    "content": {
+                        "<-": "@segmentedBar2:segment.object.{label: region, length: size}"
+                    }
+                }
+            },
+
+
+            "segmentedBar3": {
+                "prototype": "montage/ui/segmented-bar.reel",
+                "properties": {
+                    "element": {"#": "segmentedBar3"},
+                    "childrenPath": "regions",
+                    "contentController": {"@": "treeController"}
+                }
+            },
+            "treeController": {
+                "prototype": "montage/core/tree-controller",
+                "bindings": {
+                    "content": {"<-": "@test.tree"}
+                }
+            },
+            "segment3": {
+                "prototype": "montage/ui/segment.reel",
+                "properties": {
+                    "element": {"#": "segment3"}
+                },
+                "bindings": {
+                    "content": {
+                        "<-": "@segmentedBar3:segment.object.content.{label: region, length: size}"
+                    }
+                }
+            },
+
+
+            "segmentedBar4": {
+                "prototype": "montage/ui/segmented-bar.reel",
+                "properties": {
+                    "element": {"#": "segmentedBar4"},
+                    "contentController": {"@": "rangeController"},
+                    "orientation": "vertical"
+                }
+            },
+            "rangeController": {
+                "prototype": "montage/core/range-controller",
+                "bindings": {
+                    "content": {"<-": "@test.content"}
+                }
+            },
+            "segment4": {
+                "prototype": "montage/ui/segment.reel",
+                "properties": {
+                    "element": {"#": "segment4"}
+                },
+                "bindings": {
+                    "content": {
+                        "<-": "@segmentedBar4:segment.object.{label: region, length: size}"
+                    }
+                }
+            }
+        }
+    </script>
+
+</head>
+<body>
+
+<div style="width: 960px; height: 30px;">
+    <div data-montage-id="segmentedBar1" id="segmentedBar1">
+        <div data-montage-id="segment1"></div>
+    </div>
+</div>
+
+<div data-montage-id="segmentedBar2" id="segmentedBar2">
+    <div data-montage-id="segment2"></div>
+</div>
+
+<div data-montage-id="segmentedBar3" id="segmentedBar3">
+    <div data-montage-id="segment3"></div>
+</div>
+
+<div style="width: 30px; height: 960px;">
+    <div data-montage-id="segmentedBar4" id="segmentedBar4">
+        <div data-montage-id="segment4"></div>
+    </div>
+</div>
+
+</body>
+</html>
```

Comment:
nit: I'm not sure manually formatting the serialization is useful. MontageStudio formats the serialization very predictably with the @owner first (@test would be the equivalent in this case) and all the other objects serialized by key alphabetical order after that. This makes it easy to find objects, and I'd rather not have to wonder whether Studio or a human formatted a particular source's serializations before trying to find an object, especially if in the later case I have to reverse engineer that human's formatting rationale and wonder for example if the double blank lines mean something special or are just that human's idea of what beautiful code is. Also, having a serialization style different from Studio's may cause unnecessary diffs when code is passed between a Studio user and a manual editor.